pkg/sql/plan/function/operators.go

```diff
@@ -40,7 +40,7 @@ func initOperators() {
 var operators = map[int]Functions{
 	ISTRUE: {
 		Id:     ISTRUE,
-		Flag:   plan.Function_STRICT,
+		Flag:   plan.Function_PRODUCE_NO_NULL,
```

Comment:
is true does not produce null?   istrue(null) is false?   I am not sure.

Reply:
i have tested in mo and mysql, it's always false

MySQL [mo]> select null is true;
+--------------+
| null is true |
+--------------+
| false        |
+--------------+
1 row in set (0.00 sec)

MySQL [mo]> select null is false;
+---------------+
| null is false |
+---------------+
| false         |
+---------------+
1 row in set (0.00 sec)